notebook/tests/selenium/test_clipboard_multiselect.py

```diff
@@ -0,0 +1,29 @@
+"""Tests clipboard by copying, cutting and pasting multiple cells"""
+from selenium.webdriver.common.keys import Keys
+from .utils import wait_for_selector, wait_for_xpath
+
+def test_clipboard_multiselect(notebook):
+
+    # Cell contents setup
+    values = ['1', '2', '3', '4', '5a', '6b', '7c', '8d']
+    [notebook.append(i) for i in values]
```

Comment:
Pet peeve: I don't like list comprehensions used for a side effect. If you're not building a list, use a regular loop!

In this case, though, we don't need either - `notebook.extend()` takes a list.